huaweicloud/services/eps/resource_huaweicloud_enterprise_project.go

```diff
@@ -0,0 +1,206 @@
+package eps
+
+import (
+	"context"
+	"regexp"
+	"time"
+
+	"github.com/chnsz/golangsdk/openstack/eps/v1/enterpriseprojects"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+)
+
+func ResourceEnterpriseProject() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceEnterpriseProjectCreate,
+		ReadContext:   resourceEnterpriseProjectRead,
+		UpdateContext: resourceEnterpriseProjectUpdate,
+		DeleteContext: resourceEnterpriseProjectDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(5 * time.Minute),
+			Update: schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		//request and response parameters
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
according to Update function, seems `name` can be updated?

Reply:
fixed